embodiedscan/explorer.py

```diff
@@ -68,7 +71,8 @@ def __init__(self,
         self.metainfo = None
         data_list = []
         for file in self.ann_files:
-            data = mmengine.load(file)
+            with open(file, 'rb') as f:
+                data = pickle.load(f)
```

Comment:
Why not use mmengine

Reply:
Mmengine requires pytorch.
This is not necessary for just browsing statistics.